app/assets/stylesheets/custom.scss

```diff
@@ -248,4 +248,15 @@ address {
 
 .alert-list li {
   color: white;
+}
+
+//words
+
+.form-control {
+  display: block;
+  width: 100%;
+  height: calc(2.25rem + 2px);
+  padding: 0.375rem 0.75rem;
+  font-size: 1rem;
+  line-height: 1.5;
```

Comment:
Please dont target bootstrap classes, it will change everything that uses the form-control class. You can add custom class for these css.